UI/Pages/Index.cshtml

```diff
@@ -25,11 +25,12 @@
       </div>
     }
     <form method="get">
+      <span class="govuk-caption-l">National Tutoring Programme (NTP)</span>
       <h1 class="govuk-heading-l">Find a tuition partner</h1>
```

Comment:
We also need to update the title according to the specified requirement, changing it from "Find a tuition partner" to "Find a quality assured tuition partner."

Reply:
Yep, spotted this was added to the details after, have now done that and removed the links opening in new tab as requested by Chris.